test/RPCSession.cpp

```diff
@@ -339,22 +339,25 @@ Json::Value RPCSession::rpcCall(string const& _methodName, vector<string> const&
 	return result["result"];
 }
 
+string const& RPCSession::accountCreate()
```

Comment:
Trust the force you must?

Reply:
Followed the naming of the other one, `accountCreateIfNotExists`.